recipe/meta.yaml

```diff
@@ -10,15 +9,18 @@ package:
 source:
   fn: {{ pypi_name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 490cff42650733d1832728b937fe99fa1802896f5ef4d61bcf78cf7ebecb107b
 
 build:
+  number: 0
   noarch: python
-  number: 1001
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script:
+    - export PYSOUNDFILE_PLATFORM=win32
```

Comment:
This is weird for non-windows platforms

Reply:
agreed, but it's necessary so that the noarch package works on all platforms. @mgeier is fixing it upstream. https://github.com/bastibe/python-soundfile/pull/288

This fixes a longstanding issue for Windows users on conda-forge.

Reply:
Also I tested on Linux and everything seems to work fine.

Reply:
Ahh great! Maybe it would be easiest to wait for the fix and a full release then?  Alternatively, this could use some heavy documentation and instructions on when to remove

Reply:
Yeah, I can add some comments. But git blame also always works, and I documented pretty clearly what's going on up there in the comments of this PR.